src/main/java/uk/gov/ons/census/action/schedule/ActionRuleProcessor.java

```diff
@@ -183,7 +184,11 @@ private void executeFieldCases(Stream<Case> cases, ActionRule triggeredActionRul
 
   private Specification<Case> createSpecificationForActionableCases(String actionPlanId) {
     return where(isActionPlanIdEqualTo(actionPlanId))
-        .and(excludeReceiptedCases().and(excludeRefusedCases()).and(excludeAddressInvalidCases()));
+        .and(
+            excludeReceiptedCases()
+                .and(excludeRefusedCases())
```

Comment:
These `and`s are now being called on `excludeReceiptedCases()` instead of the outer `where`. Still seems to work fine but seems neater to chain them all at the outer lever?
```java
    return where(isActionPlanIdEqualTo(actionPlanId))
        .and(excludeReceiptedCases())
        .and(excludeRefusedCases())
        .and(excludeAddressInvalidCases())
        .and(excludeHiCases());
```

Reply:
Fixed.